src/python/pants/option/global_options.py

```diff
@@ -201,3 +201,5 @@ def register_options(cls, register):
     register('--subproject-roots', type=list, advanced=True, fromfile=True, default=[],
              help='Paths that correspond with build roots for any subproject that this '
                   'project depends on.')
+    register('--force', type=bool, recursive=True,
```

Comment:
OK, this expands scope a bit since this option was suggested by @benjyw specifically to invalidate test results, which are fingerprinted today but not cached. I'll dig back in to look at wiring up cache invalidation, have the now deprecated `invalidate` pass a flag to _not_ do that and post a diff.